packages/libraries/core/tests/reporting.spec.ts

```diff
@@ -272,10 +271,10 @@ test('should send data to app.graphql-hive.com/graphql by default', async () =>
   http.done();
 
   expect(logger.getLogs()).toMatchInlineSnapshot(`
-    [INF] [hive][reporting] Publish schema
-    [DBG] [hive][reporting] POST https://app.graphql-hive.com/graphql (x-request-id=xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx) Attempt (1/6)
-    [DBG] [hive][reporting] POST https://app.graphql-hive.com/graphql (x-request-id=xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx) succeeded with status 200 (666ms).
-    [INF] [hive][reporting] Published schema
+    [INF] [hive][reporting]Publish schema
+    [DBG] [hive][reporting]POST https://app.graphql-hive.com/graphql (x-request-id=xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx) Attempt (1/6)
+    [DBG] [hive][reporting]POST https://app.graphql-hive.com/graphql (x-request-id=xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx) succeeded with status 200 (666ms).
+    [INF] [hive][reporting]Published schema
```

Comment:
Hive Logger does not append a space, you have to `logger.child('[reporting] ')`.